license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/GitPropertiesProvider.java

```diff
@@ -57,13 +57,20 @@ GitLookup getGitLookup(AbstractLicenseMojo mojo, File file, Properties props) th
           String checkCommitsCountString = props.getProperty(MAX_COMMITS_LOOKUP_KEY);
           // Backwads compatibility
           if (checkCommitsCountString == null) {
-              checkCommitsCountString = props.getProperty(COPYRIGHT_LAST_YEAR_MAX_COMMITS_LOOKUP_KEY);              
+              checkCommitsCountString = props.getProperty(COPYRIGHT_LAST_YEAR_MAX_COMMITS_LOOKUP_KEY);
           }
           int checkCommitsCount = Integer.MAX_VALUE;
           if (checkCommitsCountString != null) {
             checkCommitsCountString = checkCommitsCountString.trim();
             checkCommitsCount = Integer.parseInt(checkCommitsCountString);
           }
+          String commitsToIgnoreString = props.getProperty(COMMITS_TO_IGNORE);
+          Set<String> commitsToIgnore = Collections.emptySet();
+          if (commitsToIgnoreString != null) {
+            commitsToIgnoreString = commitsToIgnoreString.trim();
+            commitsToIgnore = Arrays.stream(commitsToIgnoreString.split(","))
```

Comment:
One other suggestion.  Splitting on a comma forces users into a long one-line string here.  You could split on a non-hex digit, e.g., `[^0-9a-fA-F]+` (and also filter out empty strings after the split) which would allow flexibility such as listing commits on newlines like:
```xml
<license.git.commitsToIgnore>
  6eb93dcfc014bfd3980c1f1c43f10fd59422107e,
  ad1e59a6afc5a1a7edbdc8bcf949bb2f8682f074,
  70403fdbc424e5d2045389e685f9bb9bd696d5cd
</license.git.commitsToIgnore>
```

Reply:
By the way, the reason I'm making this suggestion is that I really think this is a cool feature and I'm eager to use it, and somewhat biased by how I want to use it. ;)

Reply:
I looked into this, I'm a bit vary to use lots of regex magic here as it is not easily testable. It would probably make sense to create a test for `GitPropertiesProvider`, but that would require factoring out the instantiation of the `GitLookup` into some kind of factory.

So for the time being, I added a `trim` operation to the streaming expression, that should basically result in the same behaviour. So technically, I believe your example above should work :-)

Reply:
> I looked into this, I'm a bit vary to use lots of regex magic here as it is not easily testable. 

Fair enough. I hit regex101 to play around with my proposal and it took a few iterations to confirm what I expected; and the added complexity of producing empty strings that you then have to filter was undesirable. I like your `trim()` solution and it (should) work with my use case.

> It would probably make sense to create a test for `GitPropertiesProvider`, but that would require factoring out the instantiation of the `GitLookup` into some kind of factory.

Yeah. I'm not fond of the existing lazy initialization but haven't had time to think of a better way to do it.

Reply:
> > It would probably make sense to create a test for `GitPropertiesProvider`, but that would require factoring out the instantiation of the `GitLookup` into some kind of factory.
> 
> Yeah. I'm not fond of the existing lazy initialization but haven't had time to think of a better way to do it.

So that is fixed in #339